solutions/core-landing-zone/setters.yaml

```diff
@@ -47,9 +47,11 @@ data:
   ##########################
   #
   # This is the project where the config controller instance is running
-  # Values can be viewed in the Project Dashboard
+  # Values can be viewed in the Project Dashboard via https://console.cloud.google.com/welcome?project=$PROJECT_ID under "Project number"
```

Comment:
might be worth adding project ID also in the comment? 
`... under "Project ID" and "Project number"`

Reply:
Alain, good point, will do, repatch eta 2-4h

Reply:
add reference to export block for PROJECT_ID in the readme
https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/blob/main/docs/landing-zone-v2/README.md#initial-organization-configuration
and management-proiject-id below this line